README.md

```diff
@@ -121,21 +121,31 @@ This is the equivalent of:
 ip addr del MY_IP dev MY_INTERFACE
 ```
 
-## Dev environment
+## Development environment
 
-To make it work in dev you might want to create some dummy interfaces:
+To make it work in dev you need to create dummy interfaces which will be
+manipulated by LinK to simulate failover.
+
+The best way is to use systemd to automate this setup:
 
-```shell
-modprobe dummy
-ip link add eth10 type dummy
-ip link set eth10 up
-ip link add eth11 type dummy
-ip link set eth11 up
-ip link add eth12 type dummy
-ip link set eth12 up
 ```
+for idx in 10 11 12 ; do
```

Comment:
suggestion: maybe explicitly mention that this code block is `bash` to make sure it's correctly colored?